src/resources/js/aggregator-fields.js

```diff
@@ -614,13 +614,6 @@ tribe_aggregator.fields = {
 			data: rows
 		};
 
-		// if eb then reverse the order of events
-		if ( is_eventbrite ) {
-			args.order = [
-				[ 1, 'desc' ]
-			];
-		}
```

Comment:
do you need to run `gulp` to minify/process this change?

Reply:
also, does this change need a readme entry?

Reply:
No gulp required. Bot will do that